### PR TITLE
Implement EU, EEA or Switzerland route for Landlord immigration check CNTD

### PR DESCRIPTION
**Pull request by ikennaokpala**
Supersedes #2720 

[Trello card](https://trello.com/c/P3y4E5sg/316-implement-eu-country-the-eea-or-switzerland-landlord-immigration-check)
## Description

This PR modifies the flow for Landlord Immigration Check, thus adding the following:
- new questions 3a, 7 and 14
- new landlord checking service outcome
## Factcheck question 3a

[Preview link](https://smart-answers-pr-2727.herokuapp.com/landlord-immigration-check/y/B11PW/yes/yes)
[GOV.UK](https://www.gov.uk/landlord-immigration-check/y/B11PW/yes/yes)

## Expected changes
- Replacement of YES/NO with EU/EEA option/route
#### Before

screen shot 2016-09-12 at 11 41 10
#### After

screen shot 2016-09-12 at 11 41 03
## Factcheck question 7

[Preview link](https://smart-answers-pr-2727.herokuapp.com/landlord-immigration-check/y/B1%201PW/yes/yes/eea)
[GOV.UK](https://www.gov.uk/landlord-immigration-check/y/B1%201PW/yes/yes/no)
## Expected changes
- Replaced former question on this route
#### Before

screen shot 2016-09-12 at 11 49 13
#### After

screen shot 2016-09-12 at 11 48 41

### CODE REVIEW

File: lib/smart_answer_flows/landlord-immigration-check.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@ def define
       status :published
```
Comment (pmanrubia) [inline lib/smart_answer_flows/landlord-immigration-check.rb]: Could you please update the commit notes with a link to the Google Docs containing the flow?

Comment (ikennaokpala) [inline lib/smart_answer_flows/landlord-immigration-check.rb]: @pmanrubia I have updated commit notes with reference and links  from document[1] and flow diagram[2] 

[1] https://docs.google.com/document/d/1PtZxv9Fy-SOUtRSDvUq7_jbqLYUBn8Pp0nCf57U2UIE/edit
[2] https://docs.google.com/drawings/d/1x2547adffe616JMZ6qnI2ST5H2f-OnnUwSZ5isn5dzM/edit

**Comment by ikennaokpala** [comment]
@pmanrubia thanks for picking up on this.. I have squashed they as discussed.. if need be to squash more.. sure thing will do so..

**Comment by pmanrubia** [comment]
According to this document about [regression tests][1], and this [section][2] in particular, I would try to make the commits more atomic by:

### Commit structure

```txt
** I understand that comments from 1) to 6) are a bit subjective,
** so it is your call to merge it if you think that your approach is a better fit.
```

1. Remove checksums file from d11ae8856096ff3b90311459a97f554b8ceb8325. 
2. Squash commits 800caf6c2e1ce94e17321e5c4c6c08afda5a3e4a and c15690e955094b392fbc033d508632b250dbec9c including detailed detailed commit notes of why the regression tests changes are correct
3. Remove checksums file from 37eba9ff229b7b8fa9c9121cf5eed79e532bb53b
4. Squash commits 8f0c92b6499e76d741f1a87c2b40c8b6aa0f7803 and a1ab94f30fce06edd84100f70a23f61d3c7c4a52 including detailed detailed commit notes of why the regression tests changes are correct
5. Remove checksums file update from e5e13ea970e2ab48348465643e045353cf070a02
6. Squash commits 83e1ef40d7bb857c680f6adad6cd1c9db5fcd243 and a1d906f7157a484e62e8dae3d380736dd1480d31 including detailed detailed commit notes of why the regression tests changes are correct
7. Squash commits 133ebb4aafdc935d90ffd3acda3c150f8d841e31, e8fa3e88d646af0814ad4bfd74e15c2061d9162a and 7a48753fe3399958dafef84a46f4907a07beab45


### Commit notes

* c3829b93021fb9802786211a6fcb8111b2d3cb5c: 

The comment `This commit also shows the affected/changes to regression test artefacts
as a result of the changes made in the previous commit.` does not apply as the artefacts are not for previous commit. 

`With the aforementioned nodes removed.` would not apply here?

* b422f8f3edb165d8c57b21507d36a31ba35469c5: 

The comment `This commit also shows the affected/changes to regression test artefacts
as a result of the changes made in the previous commit.` does not apply as the artefacts are not for previous commit. 

`With the aforementioned nodes removed.` would not apply here. 

* Please, review all commit notes if you squash them.

[1]: https://github.com/alphagov/smart-answers/blob/master/doc/regression-tests.md
[2]: https://github.com/alphagov/smart-answers/blob/master/doc/regression-tests.md#developer-workflow

**Comment by ikennaokpala** [comment]
@pmanrubia I have squashed as you requested and to this effect consolidating checksums in the process.. I hope this helps.

**Comment by pmanrubia** [comment]
Thank you @ikennaokpala 
LGTM

**Pull request merged**